delta/layers/common_layers_test.py

```diff
@@ -0,0 +1,152 @@
+# Copyright (C) 2017 Beijing Didi Infinity Technology and Development Co.,Ltd.
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Common layers test."""
+
+import tensorflow as tf
+from absl import logging
+
+import common_layers as cl
+
+class LossUtilTest(tf.test.TestCase):
+  ''' common layer unittest '''
+
+  def test_depthwise_separable_conv(self):
+    '''test depthwise separable conv'''
+    inputs = tf.random_uniform(shape=[1, 5, 5, 1], dtype=tf.float32, maxval=1.0)
+    num_pwc_filters = 3 #out filters num
+    width_multiplier = 1
+    scope = 'depthwise'
+    output = cl.depthwise_separable_conv(inputs,
+                             num_pwc_filters,
+                             width_multiplier,
+                             scope)
+    shape = [1, 5, 5, 3]
+    self.assertAllEqual(tf.shape(output), shape)
+
+
+  def test_splice_layer(self):
+    '''test splice layer'''
+    inputs = tf.reshape(tf.range(15), shape=[1, 5, 3])
+    context = [1, 3]
+    output = cl.splice_layer(inputs, 'splice', context)
+    output_true = tf.constant([[[3, 4, 5, 9, 10, 11],
```

Comment:
this unit test not true, and `https://github.com/didi/delta/blob/24ac0b3a021e4aa277557fc0b6654f86823135e8/delta/layers/common_layers.py#L84` also not work well. Please fix it, and add more details.